### PR TITLE
feat: #4013 webhook types

### PR DESCRIPTION
**Pull request by bashleigh**
# Pull request checklist

**Detail as per issue below (required):**

fixes: #4013 

## Usage

```ts

const handleEvent = (event: ReapitWebhookPayloadType) => {
  switch (event.topicId) {
    case reapitWebhookTopicEnum.APPLICATION_INSTALL:
      // event.new is AppDetailsModel
      // event.diff is Partial<AppDetailsModel>
      myParticularEvent(event)
      break;
    default:
      //event.new is any
  }
}

const myParticularEvent = (event: ReapitWebhookApplicationInstallEvent) => {
  // event.new is AppDetailsModel
}

```

### CODE REVIEW

File: packages/foundations-ts-definitions/types/webhook-schema.ts [hunk 1]
```diff
@@ -0,0 +1,306 @@
+import {
```
Comment (willmcvay) [inline packages/foundations-ts-definitions/types/webhook-schema.ts:1]: So you will need to move this to it's own directory like the traffic-schema. Reason being the types dir gets rm -rf ed by the definitions cron job so only the auto generated types can live in the types directory

File: packages/foundations-ts-definitions/types/index.ts [hunk 2]
```diff
@@ -7,3 +7,4 @@ export * from './marketplace-schema'
 export * from './payments-schema'
 // @ts-ignore
 export * from './platform-schema'
+export * from './webhook-schema'
```
Comment (willmcvay) [inline packages/foundations-ts-definitions/types/index.ts:10]: See below, you need to remove this line and add the webhook schema here https://github.com/reapit/foundations/blob/master/packages/foundations-ts-definitions/scripts/create-index-file.js#L19

**Review by willmcvay — CHANGES_REQUESTED** [review]
Sorry it's a bit janky but you just need to tweak a couple of bits, can you also add that usage thing you added to the PR to the README.md

Other than that LGTM.

File: packages/foundations-ts-definitions/types/webhook-schema.ts (hunk 1 above)
Comment (bashleigh) [reply packages/foundations-ts-definitions/types/webhook-schema.ts:1]: thought this was the case. Forgot to check!

**Review by willmcvay — APPROVED** [review]

**Pull request merged**